src/Microsoft.Azure.SignalRService/ServiceManager.cs

```diff
@@ -3,16 +3,53 @@
 
 using System;
 using System.Collections.Generic;
+using System.Linq;
 using System.Security.Claims;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.SignalR;
+using Microsoft.Extensions.DependencyInjection;
 
 namespace Microsoft.Azure.SignalRService
 {
     public class ServiceManager : IServiceManager
     {
+
+        internal ServiceManagerContext ServiceManagerContext { get; }
+
+        internal ServiceManager(ServiceManagerContext serviceManagerContext)
+        {
+            ServiceManagerContext = serviceManagerContext;
+        }
+
         public Task<IServiceHubContext> CreateHubContextAsync(string hubName)
         {
-            throw new NotImplementedException();
+            var hubInfo = new HubInfo(hubName);
+
+            var serviceCollection = new ServiceCollection();
+            serviceCollection.AddSignalRCore();
+
+            // remove default hub lifetime manager
+            var serviceDescriptor = serviceCollection.FirstOrDefault(descriptor => descriptor.ServiceType == typeof(HubLifetimeManager<>));
+            serviceCollection.Remove(serviceDescriptor);
```

Comment:
Same here, you only need to remove it once?

Reply:
Same above